src/xml_parsing.cpp

```diff
@@ -541,6 +541,21 @@ void VerifyXML(const std::string& xml_text,
           ThrowError(line_number,
                      std::string("The node <") + name + "> must have 1 or more children");
         }
+        if (name == "ReactiveSequence")
+        {
+          const std::string child_name = node->FirstChildElement()->Name();
+          const auto child_search = registered_nodes.find(child_name);
+          auto child_type = child_search->second;
+          if (child_type == NodeType::CONTROL &&
+              ((child_name == "ThreadedAction") ||
+              (child_name == "StatefulActionNode") ||
+              (child_name == "CoroActionNode"))
+             )
```

Comment:
@facontidavide this is the best i can come up with to determine if a node is asynch. Am I missing an easier way?

Reply:
of course, an entire subtree could be asynchronous... but as a first attempt to prevent people from shooting themselves in the foot, it is OK.

Reply:
Great. I added a unit test. I'll work on expanding the functionality to count # of children that are asynchronous now.

Reply:
Alright, ready for review I think.